app/assets/javascripts/discourse-common/addon/lib/raw-templates.js

```diff
@@ -50,28 +50,9 @@ export function addRawTemplate(name, template, opts = {}) {
     const message = `[${name}] hbr topic-list template overrides and connectors are deprecated. Use the value transformer \`topic-list-columns\` and other new topic-list plugin APIs instead.`;
 
     // NOTE: addRawTemplate is called too early for deprecation handlers to process this:
-    deprecated(message, {
-      since: "v3.4.0.beta3-dev",
-      id: "discourse.hbr-topic-list-overrides",
-    });
+    deprecated(message, RAW_TOPIC_LIST_DEPRECATION_OPTIONS);
 
     needsHbrTopicList(true);
-
-    let prefix;
-    if (opts.themeId) {
-      prefix = consolePrefix(null, {
-        type: "theme",
-        id: opts.themeId,
-        name: opts.themeName,
-      });
-    } else if (opts.pluginName) {
-      prefix = consolePrefix(null, {
-        type: "plugin",
-        name: opts.pluginName,
-      });
-    }
```

Comment:
how about adding those to the `message`?

Reply:
deprecate() will add them automatically based on the backtrace (`addRawTemplate` is called from theme/plugin `.js` bundles)

Reply:
ah, so that was fixed ~recently? in #30299?

Reply:
Possibly yeah! Just double-checked locally, and it's definitely working:

<img width="976" alt="SCR-20250108-kvcv" src="https://github.com/user-attachments/assets/bc27b5bb-6c0a-41d1-aad7-a2f112d04990" />